### ISSUE TITLE
ivy: failed to compile when i use renamed external library class in ng library

### ISSUE THREAD

**Issue by noxi515** [angular/angular#34592]
# 🐞 bug report

### Affected Package

I dont know.

### Is this a regression?

Yes. It works well on Angular8.

### Description

When I use an extenral library class `@microsoft/applicationinsights-web/ApplicationInsights` in angular library project, it causes compile error.
`ApplicationInsights` is originally named `Initialization` and renamed when exporting. Ivy compiler works well on this case in main application project, library project not.

```
export { IApplicationInsights, Snippet, Initialization as ApplicationInsights } from "./Initialization";
```

## 🔬 Minimal Reproduction

https://github.com/noxi515/angular-ivy-lib-issue

1. `ng new` to create new project.
2. `ng g library my-lib` to create new library.
3. `npm i -P @microsoft/applicationinsights-web`
4. Use `@microsoft/applicationinsight-web/ApplicationInsights` class in `MyLibService`
5. Use `my-lib/MyLibService` in `AppComponent`
6. `ng build my-lib --prod`
7. `ng build --prod`

## 🔥 Exception or Error
<pre><code>
ERROR in ./dist/my-lib/__ivy_ngcc__/fesm2015/my-lib.js 12:106-127
"export 'Initialization' (imported as 'ɵngcc1') was not found in '@microsoft/applicationinsights-web'
    at HarmonyImportSpecifierDependency._getErrors (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\dependencies\HarmonyImportSpecifierDependency.js:109:11)
    at HarmonyImportSpecifierDependency.getErrors (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\dependencies\HarmonyImportSpecifierDependency.js:68:16)
    at Compilation.reportDependencyErrorsAndWarnings (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compilation.js:1463:22)
    at C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compilation.js:1258:10
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:42:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\tapable\lib\Hook.js:154:20)      
    at Compilation.finish (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compilation.js:1253:28)
    at C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compiler.js:672:17
    at _done (eval at create (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:9:1)
    at eval (eval at create (C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:48:22)
    at C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compilation.js:1185:12
    at C:\Users\noxi\Projects\angular-9\angular-ivy-lib-issue\node_modules\webpack\lib\Compilation.js:1097:9
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts
</code></pre>

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 9.0.0-rc.7
Node: 12.14.0
OS: win32 x64

Angular: 9.0.0-rc.7
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.900.0-rc.7
@angular-devkit/build-angular      0.900.0-rc.7
@angular-devkit/build-ng-packagr   0.900.0-rc.7
@angular-devkit/build-optimizer    0.900.0-rc.7
@angular-devkit/build-webpack      0.900.0-rc.7
@angular-devkit/core               9.0.0-rc.7
@angular-devkit/schematics         9.0.0-rc.7
@ngtools/webpack                   9.0.0-rc.7
@schematics/angular                9.0.0-rc.7
@schematics/update                 0.900.0-rc.7
ng-packagr                         9.0.0-rc.3
rxjs                               6.5.4
typescript                         3.6.4
webpack                            4.41.2
</code></pre>

**Anything else relevant?**

**Comment by alxhub** [comment]
Should be fixed in c0c2ab30.

**Comment by noxi515** [comment]
Is this really fixed? I have same error on 9.0.0-rc.10

```
Angular CLI: 9.0.0-rc.10
Node: 12.13.0
OS: win32 x64

Angular: 9.0.0-rc.10
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.900.0-rc.10
@angular-devkit/build-angular      0.900.0-rc.10
@angular-devkit/build-ng-packagr   0.900.0-rc.10
@angular-devkit/build-optimizer    0.900.0-rc.10
@angular-devkit/build-webpack      0.900.0-rc.10
@angular-devkit/core               9.0.0-rc.10
@angular-devkit/schematics         9.0.0-rc.10
@angular/cdk                       9.0.0-rc.8
@angular/material                  9.0.0-rc.8
@ngtools/webpack                   9.0.0-rc.10
@schematics/angular                9.0.0-rc.10
@schematics/update                 0.900.0-rc.10
ng-packagr                         9.0.0-rc.7
rxjs                               6.5.4
typescript                         3.7.5
webpack                            4.41.2
```
